pushapkscript/test/test_googleplay.py

```diff
@@ -15,160 +17,179 @@
 @patch('pushapkscript.googleplay.push_apk')
 class GooglePlayTest(unittest.TestCase):
     def setUp(self):
-        self.context = MagicMock()
-        self.context.config = {
-            'google_play_accounts': {
-                'aurora': {
-                    'service_account': 'aurora_account',
-                    'certificate': '/path/to/aurora.p12',
-                },
-                'beta': {
-                    'service_account': 'beta_account',
-                    'certificate': '/path/to/beta.p12',
-                },
-                'release': {
-                    'service_account': 'release_account',
-                    'certificate': '/path/to/release.p12',
-                },
-                'dep': {
-                    'service_account': 'dummy_dep',
-                    'certificate': '/path/to/dummy_non_p12_file',
-                },
-            },
-            'taskcluster_scope_prefixes': ['project:releng:googleplay:'],
+        self.product = 'release'
+        self.task_payload = {
+            'google_play_track': 'alpha'
         }
-        self.context.task = {
-            'scopes': ['project:releng:googleplay:release'],
-            'payload': {
-                'google_play_track': 'alpha'
+
+        self.products = {
+            'aurora': {
+                'has_nightly_track': False,
+                'service_account': 'aurora_account',
+                'certificate': '/path/to/aurora.p12',
+                'skip_check_package_names': True,
+            },
+            'beta': {
+                'has_nightly_track': False,
+                'service_account': 'beta_account',
+                'certificate': '/path/to/beta.p12',
+                'skip_check_package_names': True,
+            },
+            'release': {
+                'has_nightly_track': False,
+                'service_account': 'release_account',
+                'certificate': '/path/to/release.p12',
+                'skip_check_package_names': True,
             },
+            'dep': {
+                'has_nightly_track': False,
+                'service_account': 'dummy_dep',
+                'certificate': '/path/to/dummy_non_p12_file',
+                'skip_check_package_names': True,
+                'skip_check_ordered_version_codes': True,
+                'skip_checks_fennec': True,
+            }
         }
         self.apks = [MockFile('/path/to/x86.apk'), MockFile('/path/to/arm_v15.apk')]
 
     def test_publish_config(self, mock_push_apk):
         android_products = ('aurora', 'beta', 'release')
         for android_product in android_products:
-            self.context.task['scopes'] = ['project:releng:googleplay:{}'.format(android_product)]
-            publish_to_googleplay(self.context, self.apks)
+            publish_to_googleplay(True, self.task_payload, self.products[android_product], self.apks)
```

Comment:
Nit: I'm confused by the first argument being a raw boolean. It's hard to guess what it does. How about we reorder the arguments so that `contact_google_play` becomes the last one? This would allow to write:

```py
publish_to_googleplay(self.task_payload, self.products[android_product], self.apks, contact_google_play=True) 
```